### DISCUSSION TITLE
Changing version naming

### DISCUSSION THREAD

**Discussion by Firstyear** [kanidm/kanidm#1168] [Polls]
Question for the community. The project has been really reliable/stable for a long time, and I think it might be time to shed the "alpha" tagging. We've had some comments that "alpha" indicates a lack of quality in the project and it's "too early" to adopt. Saying this, I still don't want to "freeze" the current API as I think we still want room to move about and change that

**Comment by yaleman** [comment]
Voted #2 because while the v1 API's still in flux I'd say we're still beta - unless the 1.2.0 release is going to lock it and then the project moves to a new API version for further (breaking) changes.
